examples/VerticalWithinHorizontalScrolling/Sample/GradientTableNode.mm

```diff
@@ -58,7 +58,8 @@ - (ASCellNode *)tableNode:(ASTableNode *)tableNode nodeForRowAtIndexPath:(NSInde
 {
   RandomCoreGraphicsNode *elementNode = [[RandomCoreGraphicsNode alloc] init];
   elementNode.style.preferredSize = _elementSize;
-  elementNode.indexPath = [NSIndexPath indexPathForRow:indexPath.row inSection:_pageNumber];
+  NSString *content = [[NSIndexPath indexPathForRow:indexPath.row inSection:_pageNumber] description];
+  [elementNode setContent:content];
```

Comment:
If the content is set only once, maybe pass it in as a param of the initializer and remove `-setContent:` altogether?

Reply:
fixed.